GVFS/GVFS.Common/PlaceholderListDatabase.cs

```diff
@@ -49,7 +50,7 @@ private PlaceholderListDatabase(ITracer tracer, PhysicalFileSystem fileSystem, s
         /// The EstimatedCount is "estimated" because it's simply (# adds - # deletes).  There is nothing to prevent
         /// multiple adds or deletes of the same path from being double counted
         /// </summary>
-        public int EstimatedCount { get; private set; }
+        public int Count { get; private set; }

```

Comment:
When we move to SQLite will this count be more accurate?

Reply:
Yes because it will be querying the database with 
```
SELECT COUNT(path) FROM Placeholders
```